recipe/meta.yaml

```diff
@@ -1,27 +1,26 @@
 {% set version = "3.1.5" %}
-{% set commit = "1535d223d5df0c8f6a2722796efb96c845c25405" %}
 
 package:
   name: mpfr
   version: {{ version }}
 
 source:
-  fn: mpfr-{{ version }}.tar.bz2                                                    # [unix]
-  url: http://ftp.gnu.org/gnu/mpfr/mpfr-{{ version }}.tar.gz                        # [unix]
-  sha256: f4eb5070883aee3fd8b927751ea63ff95aebe24418cde852439ce74c3dd2513c          # [unix]
-  fn: {{ commit }}.zip                                                              # [win]
-  url: https://github.com/BrianGladman/mpfr/archive/{{ commit }}.zip                # [win]
-  sha256: 050eb5f8c40e634d2681b5d84764b75bb103c35a7491e999d6a5b3a94b34f325          # [win]
+  fn: mpfr-{{ version }}.tar.bz2                                                                        # [unix]
+  url: http://ftp.gnu.org/gnu/mpfr/mpfr-{{ version }}.tar.gz                                            # [unix]
+  sha256: f4eb5070883aee3fd8b927751ea63ff95aebe24418cde852439ce74c3dd2513c                              # [unix]
+  fn: mpfr-{{ version }}-win.zip                                                                        # [win]
+  url: https://github.com/isuruf/mpfr/releases/download/{{ version }}-win/mpfr-{{ version }}-win.zip    # [win]
```

Comment:
I'm trying to make tarballs for releases as the original repo doesn't have releases and sometimes doesn't match the official mpfr release because the repo pulls in from official mpfr repo. For example, the commit used before was the one closest to the 3.1.5 official mpfr release, but the version is set as 4.0.0.dev